pallets/automation-time/src/lib.rs

```diff
@@ -328,8 +272,7 @@ pub mod pallet {
 
 	#[pallet::storage]
 	#[pallet::getter(fn get_scheduled_tasks)]
-	pub type ScheduledTasksV2<T: Config> =
```

Comment:
since you are removing this we need to remove `MaxTasksPerSlot`

Reply:
yup, working on that now. good catch